http-clients/netty-nio-client/pom.xml

```diff
@@ -60,6 +60,14 @@
 
         <!--Reactive Dependencies-->
         <dependency>
+            <groupId>io.netty</groupId>
```

Comment:
You can probably remove this. Was testing the perf of native SSL.

Reply:
Removed